tests/build.gradle

```diff
@@ -20,14 +19,10 @@ repositories {
     mavenCentral()
 }
 
-tasks.named('compileTestKotlin') {
-    dependsOn ':mockito-kotlin:jar'
-}
```

Comment:
10. Remove this as this is automatic now. And the dependency is decoupled, not direct.